.circleci/config.yml

```diff
@@ -198,6 +199,24 @@ commands:
       - slack/status:
           success_message: ":tada: deploy of <$CIRCLE_BUILD_URL|$CIRCLE_BRANCH> to << parameters.environment >> successful!"
           failure_message: ":red_circle: deploy of <$CIRCLE_BUILD_URL|$CIRCLE_BRANCH> to << parameters.environment >> failed!"
+  
+  cypress-local:
+    description: >
+      Allows for running the cypress tests locally
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 19.03.13
+      - run:
+          name: Install Docker Compose
+          command: |
+            apt install sudo
```

Comment:
maybe `apt-get install sudo`

Reply:
I think this is what i was looking for 🤦